cli/man/splinter-keygen.1.md

```diff
@@ -0,0 +1,78 @@
+% SPLINTER-KEYGEN(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**splinter-keygen** — Generates user and daemon keys for Splinter
+
+SYNOPSIS
+========
+
+**splinter keygen** \[**FLAGS**\] \[**OPTIONS**\] \[*KEY-NAME*\]
+
+DESCRIPTION
+===========
+
+This command generates secp256k1 public/private keys for Splinter.
+
+The `--system` flag generates keys for the Splinter daemon (`splinterd`) that
+are stored in `/etc/splinter/keys`.
+
+If no option is specified, this command generates user keys that are stored in
+the directory `$HOME/splinter/keys`.
+
+FLAGS
+=====
+
+`-h`, `--help`
+: Prints help information
+
+`-q`, `--quiet`
+: Decreases verbosity (the opposite of -v). When specified, only errors or
+  warnings will be output.
+
+`--system`
+: Generates system keys for `splinterd` in `/etc/splinter/keys`.
+  By default, the key files are named `root.priv` and `root.pub`, because
+  this command runs as root when the `--system` option is used.
```

Comment:
This isn't a fundamental attribute of the command - just how we normally use it. The command doesn't escalate to root, and this might suggest that it does. Perhaps change this to an example - "The key filenames are determined by username, so if running as root the filenames will be 'root.priv' and 'root.pub'."

Reply:
> ... so if running as root the filenames will be 'root.priv' and 'root.pub'."

Is there a requirement to run --system as root?  Will the system (daemon) keys be functional if you run this option as any-old-user?

Reply:
^ (above question is for @vaporos )

Reply:
You could run splinterd as another user, it does not require root permission. Within docker files, it is often running as root.

Reply:
Fixed by adding a clarification to the DESCRIPTION and removing all mentions of the command running as root.